### ISSUE TITLE
How to play?

### ISSUE THREAD

**Issue by mottosso** [team-phoenix/Phoenix#192]
Removed

**Comment by athairus** [comment]
Thank you for keeping our project on GitHub by not posting links to sites us devs totally don't go to ourselves. 

Well, for starters, launching the game when it's in the library is simple. Just double-click it!

Let me break it down. Phoenix works in two modes: portable and installed. All dev builds are portable by default. How it should work (**and this isn't actually implemented yet**) is that you import your games using the ( + ) button on the bottom left and it'll copy them to the `User Data/Games` directory, storing a path in the library relative to the executable. Right now, it stores absolute paths, making Phoenix a lousy portable app. However, it'll still work. Import your games and they should work.

The default SNES core is bsnes_mercury_balanced. If you *really* want to use snes9x, you'll have to edit systems.sqlite (in the source directory) and rebuild so it's copied to the build directory. Default cores are defined in one of the tables, I forget which.

**Comment by mottosso** [comment]
> Thank you for keeping our project on GitHub by not posting links to sites us devs totally don't go to ourselves.

Sorry, I can't tell if you are being sarcastic. Could you rephrase this?

**Comment by athairus** [comment]
Don't post links that'll get this project removed from GitHub.

**Comment by mottosso** [comment]
I have no idea to what you are referring, but I have lost interest in finding out. Thank you for not being a dick.

**Comment by athairus** [comment]
I edited your first post to remove a link to a certain site. If you're so casual about it that you forgot, then maybe you shouldn't create more issues here.
